app/src/main/java/no/usn/mob3000/data/repository/social/FriendsRepository.kt

```diff
@@ -0,0 +1,42 @@
+package no.usn.mob3000.data.repository.social
+
+import android.util.Log
+import no.usn.mob3000.data.model.social.FriendSingleDto
+import no.usn.mob3000.data.source.remote.auth.AuthDataSource
+import no.usn.mob3000.data.source.remote.social.FriendsDataSource
+import no.usn.mob3000.domain.model.social.FriendData
+import no.usn.mob3000.domain.repository.social.IFriendsRepository
+import kotlin.math.log
+/**
+* @author Hussein Abdul-Ameer
+* created on 05.11.2024
```

Comment:
Remember to conform to repository date standards. Everywhere else is using ISO8601.